diagnostic.md

```diff
@@ -5,74 +5,89 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+The back-end is where things happens behind the scenes ... it is were the processing occurs ..
 ```
 
 Which layer in the MVC pattern is used by the controller to fetch data?
 
 ```md
-// your response here
+the controller is the "C" in the model
 ```
 
 Which layer in the MVC pattern communicates with the model?
 
 ```md
-// your response here
+The Controller communicates with the model
 ```
 
 Why don't we use views in our interpretation of the MVC pattern?
 
 ```md
-// your response here
+The client uses the View
 ```
 
 What does C.R.U.D stand for?
 
 ```md
-// your response here
+Create, Read, Update, Delete
 ```
 
 In which part of the MVC pattern can we find C.R.U.D actions?
 
 ```md
-// your response here
+the CRUD occurs at the data level - our Model
 ```
 
 List at least 5 standard rails actions that C.R.U.D requests correspond to?
 
 ```md
-// your response here
+Index
+Show
+Create
+Update
+Destroy
 ```
 
 A user action fires a `GET` request for `/people/1`. Explain in detail each step
 required for data to be returned to the client. (bullet points or ordered list)
 
 ```md
-// your response here
+GET access the API,
+Routes to certain action
+Action defines what needs to be done
+View returns the ooutput
 ```
 
 What is the command to generate a new rails-api app?
 
 ```bash
-// your response here
+bundle install in the rails api repo ;-)
+it works ... I didn\'t ask why ...
```

Comment:
To generate a new rails app we would use this command:

`rails new <app_name>`